front/src/styles/scss/applications/operationalStudies/_scenario.scss

```diff
@@ -32,7 +32,6 @@
       color: var(--white);
       border: unset;
       border-radius: 4px;
-      font-size: 1.5rem;
```

Comment:
can’t use a overflow: hidden or change the height ?

Reply:
with the font-size:
![image](https://github.com/user-attachments/assets/8eafe625-9681-480c-9eb0-a38b7156685c)

without:
![image](https://github.com/user-attachments/assets/46fa2674-ac6d-4a27-b9a5-01249ecd88cc)

it's really seems better without

Reply:
With osrd the icons are now the right size, no need for the font-size here.
overflow: hidden cuts the lower part of the macro component